docs/unbrick-devnet.md

```diff
@@ -0,0 +1 @@
+# Unbrick Devnet
```

Comment:
Is this meant to be included?

Reply:
No. Accidentally created when taking notes with Pablo and Nathan on unbricking devnet. Will clean up and remove